tests/test_repair.py

```diff
@@ -53,10 +53,8 @@ def test__find_intact_sub_trees():
     assert_equal(len(obj._find_intact_sub_trees()), 2)
 
     obj = Repair(SIMPLE_PATH)
-    obj.cut_leaves = [[0, 0, 0]]
+    obj.cut_leaves = np.array([[0, 0, 0]])
     obj._fill_repair_type_map()
```

Comment:
Does it make sense to keep these 3 lines at all? We don't assert anything after them.

Reply:
Yes, I've removed them. Good point.